data/hails.txt

```diff
@@ -1098,6 +1169,11 @@ phrase "friendly navy"
 		"the biggest"
 		"the largest"
 		"the greatest"
+		"the strongest"
+		"the grandest"
+		"big brother's"
```

Comment:
1984 is more than a thousand years ago in the game world. And anyway, this doesn't fit the tone at all: the Navy neither is nor thinks of itself as an agent of "Big Brother."

Reply:
>1984 is more than a thousand years ago in the game world

The book was published in 1949 though...

>the Navy neither is nor thinks of itself as an agent of "Big Brother."

Agreed.